lib/solidus_affirm/backward_compatibility_hacks/api_template.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# This hack is needed because before v2.6 the API view that shows the order
+# was trying to call the below methods on the payment source.
+# The problem is that some sources, like the SolidusAffirm::Checkout,
+# do not have these fields defined. We are creating these methods for
+# older Solidus versions only and making them only return nil.
+first_version_with_payment_source_partials = Gem::Requirement.new('>= 2.6')
+unless first_version_with_payment_source_partials.satisfied_by?(SolidusSupport.solidus_gem_version)
```

Comment:
Personally I would move this Solidus version check into `engine.rb` and only require this file if it's needed.

Reply:
Done, thanks for the suggestion!